### ISSUE TITLE
go test file/package command results in verbose output

### ISSUE THREAD

**Issue by deltaskelta** [Microsoft/vscode-go#815]
My go settings are as follows (stock, with comments removed)

```
  "go.buildOnSave": true,
  "go.buildFlags": [],
  "go.lintOnSave": true,
  "go.lintTool": "golint",
  "go.lintFlags": [],
  "go.vetOnSave": true,
  "go.vetFlags": [],
  "go.formatTool": "goreturns",
  "go.formatFlags": [],
  "go.useDiffForFormatting": true,
  "go.useCodeSnippetsOnFunctionSuggest": false,
  "go.gopath": null,
  "go.goroot": null,
  "go.formatOnSave": true,
  "go.coverOnSave": false,
  "go.testTimeout": "30s",
  "go.gocodeAutoBuild": true,
  "go.buildTags": "",
  "go.testEnvVars": {},
  "go.autocompleteUnimportedPackages": false,
  "go.docsTool": "godoc",
  "go.toolsGopath": "",
  "go.testFlags": null,
```

The build flags and the test flags are both null so this leads me to believe that when I run the go tests I should see the non verbose output like follows.

```
PASS
ok      github.com/deltaskelta/docker-alertd/monitor    0.016s
```

but what I actually see is the verbose output, like this...(truncated)

```
=== RUN   TestGetConfJSON/1:_testing_all_the_things
=== RUN   TestGetConfJSON/2:_Testing_invalid_JSON
=== RUN   TestGetConfJSON/3:_Testing_blank_JSON
--- PASS: TestGetConfJSON (0.00s)
    --- PASS: TestGetConfJSON/1:_testing_all_the_things (0.00s)
    --- PASS: TestGetConfJSON/2:_Testing_invalid_JSON (0.00s)
    --- PASS: TestGetConfJSON/3:_Testing_blank_JSON (0.00s)
PASS
ok      github.com/deltaskelta/docker-alertd/monitor    0.041s
```

So How can I make the test output appear without the verbose output? I have poked through the code and I do not see a reason in the source that would make the `-v` flag active

**Comment by ramya-rao-a** [comment]
Can you try `go.testFlags`: []` ?

**Comment by deltaskelta** [comment]
That works. I found the line that does it. 

https://github.com/Microsoft/vscode-go/blob/547be17cb11915a34a74d68212b01d7f1296431c/src/goTest.ts#L217

This seems to include `-v` option by default. The settings imply that they would be left blank so I think either the comment should be changed to say that `-v` is included by default, or the `-v` should be omitted. What do you think? I can make a PR for either way

**Comment by ramya-rao-a** [comment]
The default value for `testFlags` is null. When the value is null, `buildFlags` and `-v` are used to keep the default experience same as before the `testFlags` feature was introduced. So that someone not touching the new `testFlags` sees no change in behavior.

Now, the more I think about it, it seems weird that `-v` is included with build flags in the default set of test flags. 

Please submit a PR to remove `-v` from the default so that there is no further confusion on this.
I'll add a note in the Changelog about test output not being verbose by default anymore

**Comment by deltaskelta** [comment]
@ramya-rao-a #817

**Comment by ramya-rao-a** [comment]
Fixed with #817

**Comment by ramya-rao-a** [comment]
Fixed in the latest update (0.6.54) to the Go extension.
